librz/main/rz-diff.c

```diff
@@ -1770,8 +1770,8 @@ static void diff_graph_function_detail_as_json(const char *object_name, RzAnalys
 	free(fcn_name);
 	pj_kn(pj, "offset", fcn->addr);
 	pj_ki(pj, "ninstr", fcn->ninstr);
-	pj_kn(pj, "nargs", rz_analysis_var_count_total(fcn, RZ_ANALYSIS_VAR_TYPE_ARGUMENT));
-	pj_kn(pj, "nlocals", rz_analysis_var_count_total(fcn, RZ_ANALYSIS_VAR_TYPE_LOCAL));
+	pj_kn(pj, "nargs", rz_analysis_var_count(fcn, true));
```

Comment:
What about providing helper functions `rz_analysis_var_count()` and `rz_analysis_arg_count()` instead of the second argument?

Reply:
Yeah agreed, it looks more self-explanatory that way.